src/components/ShowAllJobs/ShowAllJobs.tsx

```diff
@@ -1,13 +1,20 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
 import { DispatchType, RootState } from "../../redux/store/store";
 import { getAllJobs } from "../../redux/reducer/jobs";
 import { useDispatch, useSelector } from "react-redux";
 import Loading from "../Loading/Loading";
 import { JobModal } from "../../type/type";
 import JobTicket from "../Job/JobTicket";
 import "./showAllJobs.scss";
+import Pagination from "../Pagination/Pagination";
 
 const ShowAllJobs = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [jobsPerPage] = useState<number>(10);
```

Comment:
Why use state here?

Reply:
I use state to easy set value of jobsPerPage in case scalable application